utils/build/run-driver-win.cmd

```diff
@@ -1,4 +1,6 @@
 @ECHO OFF
 SETLOCAL
-IF %PLAYWRIGHT_NODEJS_PATH%x == x SET PLAYWRIGHT_NODEJS_PATH="%~dp0\node.exe"
-"%PLAYWRIGHT_NODEJS_PATH%" "%~dp0\package\lib\cli\cli.js" %*
+if not defined PLAYWRIGHT_NODEJS_PATH (
```

Comment:
nit: use all CAPITAL letters for the keywords?

Reply:
microsoft does use lowercase in the docs, so it seems better like that: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/if